earn/src/components/portfolio/modal/WithdrawModal.tsx

```diff
@@ -83,9 +83,8 @@ export default function WithdrawModal(props: WithdrawModalProps) {
   const account = useAccount();
 
   const filteredPairs = lendingPairs.filter((p) => doesLendingPairContainToken(p, selectedToken));
-  if (filteredPairs.length === 0) throw new Error(`${selectedToken.symbol} isn't part of any lending pair`);
-  const selectedPair = filteredPairs.at(selectedPairIdx) ?? filteredPairs[0];
-  const lender = selectedPair.token0.equals(selectedToken) ? selectedPair.kitty0 : selectedPair.kitty1;
+  const selectedPair = filteredPairs.at(selectedPairIdx);
+  const lender = selectedPair?.token0.equals(selectedToken) ? selectedPair.kitty0 : selectedPair?.kitty1;
   const amount = GN.fromDecimalString(inputValue[0] || '0', selectedToken.decimals);
```

Comment:
Ideally wouldn't have had to change this file, but there was some bad code in here from last time I touched it